_posts/2016-09-22-ignobel.md

```diff
@@ -0,0 +1,50 @@
+* [When zombies attack: mathematical modelling of an outbreak of zombie infection](https://www.math.upenn.edu/~ted/203S10/Projects/Zombies/Zombies.pdf) (Related: [A mathematical model of Bieber Fever: the most infectious disease of our time?](http://mysite.science.uottawa.ca/rsmith43/BieberFever.pdf))
+* [The ethics of human-chicken relationships in video games: the origins of the digital chicken](http://dl.acm.org/citation.cfm?id=2874254) (chicken-related research has scored at least four Ig Nobels so far)
+* [Misperceiving bullshit as profound is associated with favorable views of Cruz, Rubio, Trump and conservatism](http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0153419)
+* [Spider behaviors include oral sexual encounters](http://www.nature.com/articles/srep25128)
+* [Are “armchair socialists” still sitting? Cross sectional study of political affiliation and physical activity](http://www.bmj.com/content/349/bmj.g7073)
+* [Endocannabinoid signaling enhances visual responses through modulation of intracellular chloride levels in retinal ganglion cells](https://elifesciences.org/content/5/e15932) (as in, pot improves tadpole vision)
+* [Searching for Mr. Hyde: A five-factor approach to characterizing “types of drunks”](http://www.tandfonline.com/doi/full/10.3109/16066359.2015.1029920)
+* [A psychological formula for explaining the 'missing socks phenomenon'](http://mentalfloss.com/article/83809/can-washing-machines-really-eat-socks)
+* [Affective forecasting in an orangutan: predicting the hedonic outcome of novel juice mixes](http://www.ncbi.nlm.nih.gov/pubmed/27515937)
+
+
+<blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Must be an IgNobel prize waiting for this 5-year study to see if cats copy their owners&#39; accents.  <a href="https://t.co/q93Z4XhVs2">https://t.co/q93Z4XhVs2</a></p>&mdash; Jamie McMillan (@orchid_b) <a href="https://twitter.com/orchid_b/status/722377493223747584">April 19, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+A good rule of thumb is that studying animals or quirky behavior is likely to yield a result that "makes people laugh, and then makes them think" ([levitating frogs](http://www.ru.nl/hfml/research/levitation/diamagnetic/), anyone?).
+But there are still plenty of wild and wacky things that just haven't been studied yet, so here are a few ideas for which I couldn't find scholarly literature or historical information.
+Perhaps some enterprising scientist would like to take a crack at these?
+And if any of these questions already have solid research-based answers that my cursory Googling overlooked, I want to hear about it!
+
+* Why do you always get into the slowest line at the grocery store?
+* Why are credit cards, IDs and business cards all the same size? [There's an international standard](https://en.wikipedia.org/wiki/ISO/IEC_7810), but how did it come about?
```

Comment:
You didn't fix the serial comma. It's only bothering me because it's inconsistent with the second par in your piece.